build/workspace.bzl

```diff
@@ -48,6 +55,7 @@ def release_dependencies():
     cni_tarballs()
     cri_tarballs()
     debian_image_dependencies()
+    etcd_tarballs()
```

Comment:
this technically isn't a "release dependency" - I think it's only used for the integration test - but I guess it doesn't really matter.